### DISCUSSION TITLE
[Node Operator Question] Optimism sequencer location

### DISCUSSION THREAD

**Discussion by Rogalek** [ethereum-optimism/developers#283] [Miscellaneous] [answered]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hello, I hope you are all doing great.

I am building small application based on optimism blockchain and I wanted to be my application up to date as fast as possible.
I wonder if there an option to get information in which zone I should deploy my application to be really close to the optimism sequencer. I mean us-east-1, us-east-2 etc

I just want reduce latency problem.

Thank you in advance.

**Comment by imtipi** [comment]
cc @sbvegan

**Comment by hash1go** [comment]
You can perform a forward DNS lookup from the sequencer's FQDN to obtain its IP address and then execute a ping command to compare response times. However, it's possible that something like a reverse proxy might respond instead.

By the way, why do you want to deploy close to the sequencer? Normally in dApps, end-users submit transactions to an RPC endpoint, and then those transactions are transmitted to the mempool from that node. Can you explain the background behind this?

**Comment by Rogalek** [comment]
@hash1go yes i know that I deploy some code in every region and try to ping and check which region will be the fastest but I thought easier way is just to ask developers of the optimism, because they probably know that and maybe they can share that knowledge with me?

To be precise I am asking about RPC https://mainnet-sequencer.optimism.io/

**Reply by hash1go** [reply]
Hello. Thank you for sharing that.
It seems that the OP team has not disclosed much about how they operate the sequencer.

**Reply by Rogalek** [reply]
so I cannot get that information here?

**Reply by hash1go** [reply]
Since information not written in the documentation is not available, it is unlikely that detailed information about the location of the sequencer can be obtained.

**Comment by sbvegan** [comment, marked as answer]
Sorry for the delay here, we don't publicly disclose this information. You'll have to try out different regions and figure out what works best.
